pkg/reconciler/reconciler.go

```diff
@@ -254,6 +248,15 @@ func Run(opts Options) {
 			Remediator: rem,
 		},
 	}
+	// Only instantiate the converter when the webhook is enabled because the
+	// instantiation pulls fresh schemas from the openapi discovery endpoint.
+	if opts.WebhookEnabled {
+		parseOpts.Converter, err = declared.NewValueConverter(discoveryClient)
```

Comment:
It looks like this line will be hit now, since the Converter will be `nil`: https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/validate/raw/hydrate/declared_field_hydrator.go#L41

Assuming this is the intent, the log message should be updated

Reply:
Nope. The converter would be nil only when the webhook is disabled. When webhook is disabled, `DeclaredFields` won't be invoked: https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/validate/raw/raw.go#L83.

Reply:
Ack, sorry I missed that

Reply:
FYI, this also needs to go into the 1.18 release as without this, OpenAPI will still be loaded at the instantiation time.

Sorry for not capture this earlier.

This change has been verified with memory usage reduction.